MANIFEST.in

```diff
@@ -0,0 +1,12 @@
+include *.bat
+include *.py
+include *.txt
+include .coveragerc
+include buildout.cfg
+include jenkins
+include tox.ini
+recursive-include docs *.bat
+recursive-include docs *.py
+recursive-include docs *.txt
+recursive-include docs Makefile
+recursive-include mailinglogger *.txt
```

Comment:
Why is this needed? See the mush project above, I think these would be a better way to go:
https://github.com/Simplistix/mush/blob/master/setup.py#L33
https://github.com/Simplistix/mush/blob/master/setup.py#L37

nb: setuptools-git ;-)

The setup.cfg in mush is probably also a good starting point...